swarm/network/pss.go

```diff
@@ -189,7 +188,7 @@ func (self *Pss) AddPeer(p *p2p.Peer, addr pot.Address, protocall adapters.Proto
 	self.addPeerTopic(addr, topic, rw)
```

Comment:
If we put it on Pss here, we can use the MsgReadWriter interface for the peer, making it even less opiniated